### ISSUE TITLE
Cep aberto

### ISSUE THREAD

**Issue by chrisemke** [BrasilAPI/cep-promise#201]
Existe alguma previsão para adicionar a API do [CEP aberto](https://cepaberto.com/) dentro do buscador de ceps de vocês? Usamos a API do CEP aberto na empresa onde trabalho, acredito que se adicionassem poderiamos dar uma ajuda também. PS: estou postando essa issue porque não encontrei integração com a API V3 deles. Seria de grande ajuda isso, obrigado.

**Comment by lucianopf** [comment]
Ihh, olha que massa, tenho poderes de mover issues de projetos dentro de uma org! 😱 😬

**Comment by williamtorres1** [comment]
> Na época o único parâmetro esperado da chamada era o cep, agora que temos uma configuração talvez faça sentido permitir esse provedor contando que o cliente passe sua chave

Concordo, se o usuário quiser utilizar o serviço do `CepAberto`, ele passa também as configurações de acesso, assim a lib abrange mais um serviço e não corre riscos de ser banido.

**Comment by victorfernandesraton** [comment]
> > Na época o único parâmetro esperado da chamada era o cep, agora que temos uma configuração talvez faça sentido permitir esse provedor contando que o cliente passe sua chave
> 
> Concordo, se o usuário quiser utilizar o serviço do `CepAberto`, ele passa também as configurações de acesso, assim a lib abrange mais um serviço e não corre riscos de ser banido.

pensando nessa possibilidade eu fiz a lib cep-any com zero dependencia e com a possibilidade de usar ou não os provedores "padrão" como você mesmo implementar um. 

https://github.com/victorfernandesraton/cep-any
